test/test_tree_2d_eulermulti.jl

```diff
@@ -24,7 +24,11 @@ EXAMPLES_DIR = pkgdir(Trixi, "examples", "tree_2d_dgsem")
       l2   = [81.52845664909304, 2.5455678559421346, 63229.190712645846, 0.19929478404550321, 0.011068604228443425],
       linf = [249.21708417382013, 40.33299887640794, 174205.0118831558, 0.6881458768113586, 0.11274401158173972],
       initial_refinement_level = 3,
-      tspan = (0.0, 0.001))
+      tspan = (0.0, 0.001),
+      output_directory="out")
+      file = "out/deviations.txt"
+      lines = readlines(file)
+      @assert lines[end] == "15, 0.0009595796113231045, 0.0, 0.0"
```

Comment:
I would change this to something like
```suggestion
      @assert startswith(lines[end], "15")
```
Otherwise you get a failed test each time that floating point number changes.

Other than that, I think it would be good if you would delete the `out/deviations.txt` file right before the test to ensure it is not a leftover from a previous test.

Reply:
Since I only check the last line of the file, it shouldn't make a difference to delete the file before the test. Should I still delete it?

Reply:
No, what I mean is: If the file was created by a previous test, and another test execution fails to create the file, it will not be noticed. That's why I'd delete it before the testset just to be sure that it is really created in the current test invocation

Reply:
Ah true, good point. So, you mean like this, right?
```
@trixi_testset "elixir_eulermulti_shock_bubble_shockcapturing_subcell_positivity.jl" begin
    rm("out/deviations.txt", force=true)
    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_eulermulti_shock_bubble_shockcapturing_subcell_positivity.jl"),
      l2   = [81.52845664909304, 2.5455678559421346, 63229.190712645846, 0.19929478404550321, 0.011068604228443425],
      linf = [249.21708417382013, 40.33299887640794, 174205.0118831558, 0.6881458768113586, 0.11274401158173972],
      initial_refinement_level = 3,
      tspan = (0.0, 0.001),
      output_directory="out")
      file = "out/deviations.txt"
      lines = readlines(file)
      @assert startswith(lines[end], "15")
  end
```

Reply:
Thanks

Reply:
Okay. Actually, if I delete the file before anyway, I also could check the first line like
`@assert lines[1] == "# iter, simu_time, rho1_min, rho2_min"`.
Then, we also have more tested than just the existence of the file.

Reply:
I just realized: you should not use `@assert` but `@test` for a proper test.

Also, I would just check both, i.e., both the first line for the header info and the last line for some actual data.